styles/sass/main.scss

```diff
@@ -1,4 +1,5 @@
 @import 'base/link';
+@import 'base/mixins';
```

Comment:
this looks like a hack :D 
mixnis should be imported in files which require it

Reply:
You could explain me it next time :)